java/hiro-client/src/main/java/co/arago/hiro/client/rest/DefaultWebSocketClient.java

```diff
@@ -60,6 +61,19 @@
          */
         private boolean isReconnecting;
```

Comment:
1. would make it volatile
2. it is never set to true?

Reply:
This flag is set by connect(...) connector. See 
https://github.com/arago/hiro-clients/blob/98042cff36f0c4f393b5b85b60029534d99a1950/java/hiro-client/src/main/java/co/arago/hiro/client/rest/DefaultWebSocketClient.java#L273
Will add the "volatile".